api/src/Validator/AssertHasAtLeastOneManager.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Validator;
```

Comment:
Da dieser Validator nur ganz spezifisch für CampCollaborations Sinn macht, wie wärs wenn du ihn in den Namespace App\Validator\CampCollaboration machst?

Reply:
Kann man machen, ich hätte es nicht wegen der momentanen usage in einen namespace getan.
Bin da aber emotionslos.

Reply:
Bisher haben wir das mit anderen spezifischen Validierungen für ColumnLayout und ContentNode mal so angefangen. Also wenn du nichts konkret dagegen hast würde ich es in den App\Validator\CampCollaboration verschieben (bzw. App\Validator\Camp je nach Diskussion unten).